pyalgotrading/algobulls/api.py

```diff
@@ -150,11 +156,12 @@ def create_strategy(self, strategy_name: str, strategy_details: str, abc_version
             print('Fail.')
             print(f'{ex.get_error_type()}: {ex.response}')
 
-    def update_strategy(self, strategy_name: str, strategy_details: str, abc_version: str) -> dict:
+    def update_strategy(self, strategy_code: str, strategy_name: str, strategy_details: str, abc_version: str) -> dict:
         """
```

Comment:
this docstring format is not consistent everywhere

- some methods dont have endpoint mentioned, (but they have endpoints implmented)
- some methods use the term `Return`, while some use `Returns`

can we standardize this and make the docstring consistent everywhere?